sqlite3/lib/src/api/statement.dart

```diff
@@ -23,6 +23,8 @@ abstract class PreparedStatement {
   /// [SqliteException] will be thrown.
   void execute([List<Object?> parameters = const <Object>[]]);
 
+  void executeMap([Map<String, Object?> parameters = const {}]);
```

Comment:
I think we can make `executeMap` required here (just remove the default value). If a user explicitly types `.executeMap`, they can also provide the map.

```suggestion
  void executeMap(Map<String, Object?> parameters);
```